java/xtreemfs-servers/src/test/java/org/xtreemfs/utils/ScrubberTest.java

```diff
@@ -40,9 +40,9 @@
 import org.xtreemfs.pbrpc.generatedinterfaces.GlobalTypes.SYSTEM_V_FCNTL;
 import org.xtreemfs.pbrpc.generatedinterfaces.MRC.DirectoryEntries;
 import org.xtreemfs.pbrpc.generatedinterfaces.MRC.Stat;
-import org.xtreemfs.test.SetupUtils;
-import org.xtreemfs.test.TestEnvironment;
-import org.xtreemfs.test.TestHelper;
+import org.xtreemfs.SetupUtils;
+import org.xtreemfs.TestEnvironment;
+import org.xtreemfs.TestHelper;
 import org.xtreemfs.utils.xtfs_scrub.xtfs_scrub;
 
 public class ScrubberTest {
```

Comment:
Looking at the wildcard patterns in `junit_tests_*.sh`, this test is never run, because `org.xtreemfs.utils.**` is never included.